### DISCUSSION TITLE
[Feature] QR Code Generation for User-shared Links

### DISCUSSION THREAD

**Discussion by neozhu** [immich-app/immich#2868] [Feature Request]
### The feature

I would like to propose a new feature for the project that aims to enhance the sharing and accessibility of user-shared links by providing a QR code generation functionality. users can share links in a more user-friendly and accessible manner.

Benefits:

1. Convenient sharing: QR codes provide an alternative sharing method alongside textual links. Users can choose to share the QR code when it's more convenient, particularly in situations where typing or copying the link is not feasible.
2. Easy access: QR codes eliminate the need for manual input or copying of links. Recipients can simply scan the QR code to access the content quickly, making it especially useful for mobile devices.
3. Improved user experience: The inclusion of QR codes simplifies the sharing process and enhances the overall user experience, as it reduces the effort required to access shared links.

https://github.com/immich-app/immich/assets/1549611/4ed974ff-2a50-4c57-99b2-f6b8ff9bc6ac

https://github.com/immich-app/immich/assets/1549611/f38fe700-a17c-4354-bba2-c191c982d885

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by Bykow** [comment]
Would it be possible for this feature to be implemented in the mobile apps as well ?
